server/controllers/UserController.js

```diff
@@ -1,124 +1,154 @@
-import parcels from '../models/parcels';
-import users from '../models/users';
-import userValidator from '../validators/userValidator';
-
-const validateInput = userValidator;
-
+import pool from '../databaseConnection/databaseConnection';
+import Password from '../helpers/cryptPassword';
 
 class UserController {
   static deleteUser(req, res) {
-    const { id } = req.params;
-    const user = users.find(element => element.userId === parseInt(id, 10));
-    if (!user) {
-      return res.status(404).json({
-        success: false,
-        error: 'The User with the given id was not found.'
+    const id = parseInt(req.params.id, 10);
+    pool.query('DELETE FROM users WHERE id = $1', [id],
+      (error, results) => {
+        if (error) {
+          return res.status(500).json({
+            status: 500,
+            success: false,
+            error: 'Unable to delete user Account'
+          });
+        }
+        return res.status(200).json({
+          status: 200,
+          message: 'User Account successfully deleted',
+          data: results.rows[0]

```

Comment:
Why is this index set to 0 by default?

Reply:
It was to get the first element of the result.rows array. I have taken note of that. Thanks.